agents/a2a_framework.py

```
@@ -2,260 +2,254 @@
 # Enables autonomous agents to negotiate, collaborate, and share context
 
 import asyncio
-import json
-from typing import Dict, List, Any, Callable
+from typing import Dict, List, Callable
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)